frontend/src/components/EditPost.js

```diff
@@ -5,12 +5,25 @@ import { connect } from 'react-redux'
 import { addPost, removePost, editPost, postPost } from '../actions'
 
 class EditPost extends Component {
+    postPost(post) {
+        console.log(JSON.stringify(post))
+        fetch("http://localhost:3001/posts", {
+          headers: {
+            'Authorization': 'whatever-you-want',
+            'Content-Type': 'application/json'
+          },
+          method: 'POST',
+          body: JSON.stringify(post)
+        })
+        .then(function(response) {
+          return response.json()
+        })
+    };
```

Comment:
The biggest issue was that you were using the "data" field instead of the "body" field to send data. So when i logged in the server what was being sent it was all "undefined"s which was why the server was returning a 200 ok but the post wasn't showing up because the view was filtering out undefined posts.